docs/modules/control/move_to_a_pose_control/move_to_a_pose_control.rst

```diff
@@ -1,57 +1,109 @@
+
+Move to a Pose Control
 ----------------------
 
-This is a simulation of moving to a pose control.
+In this section, we present the logic of PathFinderController that drives a car from a start pose (x, y, theta) to a goal pose. A simulation of moving to a pose control is presented below.
 
 .. image:: https://github.com/AtsushiSakai/PythonRoboticsGifs/raw/master/PathTracking/move_to_pose/animation.gif
 
 
+
 PathFinderController class
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Constructor
 ~~~~~~~~~~~
 
 .. code-block:: ipython3
+
    PathFinderController(Kp_rho, Kp_alpha, Kp_beta)
 
 Constructs an instantiate of the PathFinderController for navigating a 3-DOF wheeled robot on a 2D plane.
 
 Parameters:
-- Kp_rho: The linear velocity gain to translate the robot along a line towards the goal
-- Kp_alpha: The angular velocity gain to rotate the robot towards the goal
-- Kp_beta: The offset angular velocity gain accounting for smooth merging to the goal angle (i.e., it helps the robot heading to be parallel to the target angle.)
+
+- | **Kp_rho** : The linear velocity gain to translate the robot along a line towards the goal
+- | **Kp_alpha** : The angular velocity gain to rotate the robot towards the goal
+- | **Kp_beta** : The offset angular velocity gain accounting for smooth merging to the goal angle (i.e., it helps the robot heading to be parallel to the target angle.)
 
 
 Member function(s)
 ~~~~~~~~~~~~~~~~~~
 
 .. code-block:: ipython3
+
    calc_control_command(x_diff, y_diff, theta, theta_goal)
 
 Returns the control command for the linear and angular velocities as well as the distance to goal
 
 Parameters:
-- x_diff : The position of target with respect to current robot position in x direction
-- y_diff : The position of target with respect to current robot position in y direction
-- theta : The current heading angle of robot with respect to x axis
-- theta_goal : The target angle of robot with respect to x axis
+
+- | **x_diff** : The position of target with respect to current robot position in x direction
+- | **y_diff** : The position of target with respect to current robot position in y direction
+- | **theta** : The current heading angle of robot with respect to x axis
+- | **theta_goal** : The target angle of robot with respect to x axis
 
 Returns:
-- rho : The distance between the robot and the goal position
-- v : Command linear velocity
-- w : Command angular velocity
+
+- | **rho** : The distance between the robot and the goal position
+- | **v** : Command linear velocity
+- | **w** : Command angular velocity
+
+How does the Algorithm Work
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The distance between the robot and the goal position, :math:`\rho`, is computed as
+
+.. math::
+ \rho = \sqrt{(x_{robot} - x_{target})^2 + (y_{robot} - y_{target})^2}.
+
+The distance :math:`\rho` is used to determine the robot speed. The idea is to slow down the robot as it gets closer to the target.
+
+.. math::
+ v = K_P{_\rho} \times \rho\qquad (Eq. 1)
```

Comment:
Cloud you use `:label:` for math labeling? This doc might help you: [doc](https://github.com/AtsushiSakai/PythonRobotics/blob/master/docs/modules/path_planning/quintic_polynomials_planner/quintic_polynomials_planner.rst)
<img width="442" alt="SS 2022-01-20 at 21 23 32" src="https://user-images.githubusercontent.com/3813847/150338065-5a7ded78-aa42-47ee-8c53-a3967b7d7677.png">

Reply:
Thanks, that helped a lot!